src/main/kotlin/pt/up/fe/ni/website/backend/dto/auth/ChangePasswordDto.kt

```diff
@@ -1,6 +1,11 @@
 package pt.up.fe.ni.website.backend.dto.auth
 
+import jakarta.validation.constraints.Size
+import pt.up.fe.ni.website.backend.model.constants.AccountConstants as Constants
+
 data class ChangePasswordDto(
     val oldPassword: String,
+
+    @field:Size(min = Constants.Password.minSize, max = Constants.Password.maxSize)
```

Comment:
@DoStini the maximum size is also used when creating an account. I believe this was done to prevent ridiculously long passwords but, now that I think about it, we only store the hash so there's no point. I believe it's fine to use the default value of Integer.MAX_VALUE here and in the account model